README.md

```diff
@@ -37,14 +38,14 @@ This project is currently on beta and some features _(such as listing the existi
 For this extension to work properly, you need to have the Twilio CLI and Twilio Serverless plugin installed in your computer. Additionally, having the `code` command installed in your PATH allows the extension to open VSCode after a new project is created.
 
 ### Installing the Twilio CLI
-Run `npm install twilio-cli -g` in your terminal.
+Run `npm install twilio-cli -g` in your terminal.(Using sudo for permission issues)
```

Comment:
Thanks for adding this line, though if we're gonna include this explanation, it should be a bit clearer, something along the lines of "If you run into permission issues while doing this, running the command using `sudo` might help.". 

Also, do you mind adding a space between the period and the parenthesis?

Reply:
Okay doing it right now. Thank you for the review